libraries/lib-datahandler/src/main/resources/sw360.properties

```diff
@@ -43,5 +43,9 @@
 ## first byte of the response is available.
 #backend.timeout.read = 600000
 
+## These two properties used to set max message size and frame size of thrift transport
+#backend.thrift.max.message.size=
```

Comment:
Please include the default size(may be as comment) so the user can know the what is the current value and to waht value that can update

Reply:
included default size